rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
I'm somewhat confused what's the situation around clippy:
 - I don't see `components = ["clippy"]` here (is it not needed?)
 - I see `rustup component add clippy` removed from manual instructions
 - however, I ***still do*** see `rustup component add clippy` ***present*** in `.github/workflows/gui.yml`

I didn't test it, so maybe it works, just thought I'd ask if we're sure this is consistent?

Reply:
@akavel You are right, thank you for catching this! Should be fixed now.

Reply:
sooo... still, what about the clippy entry in `.github/workflows/gui.yml`? 😅